tasks/install.yml

```diff
@@ -1,23 +1,36 @@
+- name: Create mtg user
+  user:
+    name: "{{ mtg_user }}"
+    shell: /bin/false
+    create_home: yes
+    home: /opt/mtg
+    system: yes
+
 - name: Create mtg directories
   file:
     path: "{{ item }}"
+    owner: "{{ mtg_user }}"
+    group: "{{ mtg_user }}"
     state: directory
   loop:
     - /opt/mtg
     - /etc/mtg
 
 - name: Download binary
   get_url:
-    url: "https://github.com/9seconds/mtg/releases/download/v{{ mtg_version }}/mtg-linux-{{ mtg_architecture | default('amd64') }}"
+    url: "https://github.com/9seconds/mtg/releases/download/{{ mtg_version }}/mtg-linux-{{ mtg_architecture | default('amd64') }}"
```

Comment:
this was temporary "fix" since author switched from "1.0.0" to "v1.0.0" tags. Probably need to ask him which he will use in future

Reply:
I changed it not only here, but in defaults too. This makes version format in mtg_version, in /etc/mtg/VERSION, and on upstream the same, so this should work with any format, just set correct one in var.